.golangci.example.yml

```diff
@@ -145,9 +145,8 @@ linters-settings:
   forbidigo:
     # Forbid the following identifiers
     forbid:
-      - ^fmt\.Errorf$ # consider errors.Errorf in github.com/pkg/errors
-      - ^fmt\.Print.*$ # too much log noise
-      - ^ginkgo\.F[A-Z].*$ # these are used just for local development
+      - ^print.*$ # Identifiers are written using regexp.
+      - 'fmt\.Print.*'
```

Comment:
```suggestion
    # Forbid the following identifiers (Identifier are written using regexp):
    forbid:
      - ^print.*$ 
      - 'fmt\.Print.*'
```

Reply:
super minor nit.

Reply:
@butuzov your suggestion is not based on the right (HEAD) commit